Unable to start project

### Issue and Steps to Reproduce

Clone the repository.

First, npm install.

Then, npm start

#### Versions

0.1.0 (current "master" branch)

#### Screenshots

#### Expected

The application is displayed.

#### Actual

Unable to launch project :

prototype.js:1 Uncaught Error: Cannot find module './locale'
    at webpackMissingModule (prototype.js:1)
    at Module../node_modules/moment/src/lib/moment/prototype.js (prototype.js:1)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Module../node_modules/moment/src/lib/moment/moment.js (moment.js:1)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Module../node_modules/moment/src/moment.js (moment.js:1)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Module../node_modules/react-datepicker/es/index.js (index.js:1)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Object../node_modules/@axa-fr/react-toolkit-form-input-date/dist/CustomDate.js (CustomDate.js:23)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Object../node_modules/@axa-fr/react-toolkit-form-input-date/dist/DateInput.js (DateInput.js:27)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Object../node_modules/@axa-fr/react-toolkit-form-input-date/dist/index.js (index.js:7)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Object../node_modules/@axa-fr/react-toolkit-all/dist/index.js (index.js:22)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Module../src/layout/Header/Header.js (Header.container.js:9)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Module../src/layout/Header/index.js (index.js:1)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Module../src/layout/App/App.js (index.js:14)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Module../src/layout/App/index.js (index.js:1)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Module../src/index.js (reducer.js:16)
    at __webpack_require__ (bootstrap:781)
    at fn (bootstrap:149)
    at Object.0 (index.js:1)
    at __webpack_require__ (bootstrap:781)
    at checkDeferredModules (bootstrap:45)
    at Array.webpackJsonpCallback [as push] (bootstrap:32)
    at main.chunk.js:1


#### Additional Details

- Installed packages

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "jest-fetch-mock": "^2.1.2",
     "jest-sonar-reporter": "2.0.0",
     "lodash": "^4.17.11",
-    "moment": "^2.24.0",
+    "moment": "2.24.0",
     "node-sass": "^4.11.0",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",

